scheduler/crates/api_structs/src/event/api.rs

```diff
@@ -106,7 +104,6 @@ pub mod get_events_by_meta {
     use super::*;
 
     #[derive(Deserialize)]
-    #[serde(rename_all = "camelCase")]
```

Comment:
Why do you want to remove this? Maybe in the future there will be a new field and we might forget to put back the `rename_all`

Reply:
Yes, the future yes.
I would say we will add it in the future.
For now it will add some extra useless cycles for serde.

Reply:
Actually, the cast to CamelCase is not so much: https://github.com/serde-rs/serde/blob/ce0844b9ecc32377b5e4545d759d385a8c46bc6a/serde_derive/src/internals/case.rs#L67

I'm OK to revert it back

Reply:
Yup, I think we should aim for correctness over performance, and it is very easy to forget to ass `rename_all` back. I will accept PR when it is reverted :)

Reply:
sure, np, reverted.
also added this to some forgotten places :D